### ISSUE TITLE
logits scaling affect rollout_actor_probs_pearson_corr metrics

### ISSUE THREAD

**Issue by Lokiscripter** [volcengine/verl#4162]
### Information

- [x] The official example scripts
- [x] My own modified scripts

### Tasks

- [x] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

When using VLLM for rollout, the returned rollout_log_probs are computed logits that are not scaled by temperature. If scaling is applied on the training side, it will lead to an incorrect value for the metric "rollout_actor_probs_pearson_corr." Generally, a correlation coefficient closer to 1 is considered to indicate better alignment between training and inference.

Using FSDP also has similar issues.

```Python
# verl/workers/actor/megatron_actor.py
                def logits_processor(logits, label, label_mask):
                    assert logits.shape[:2] == label.shape[:2]
                    assert label.shape == label_mask.shape
                    logits.div_(temperature) #### this affect rollout_actor_probs_pearson_corr
                    ret = {}
                    if calculate_entropy:
                        logits_bak = logits.clone()
                        logger.warning_once(
                            "For memory-efficient computation, enable fused kernels via "
                            "`actor_rollout_ref.model.use_fused_kernels=True`. "
                            "The current `clone()` operation ensures correctness but increases memory usage."
                        )
                        entropy = vocab_parallel_entropy(logits)
                        ret["entropy"] = entropy
                    else:
                        logits_bak = logits
                    log_probs = vocab_parallel_log_probs_from_logits(logits_bak, label)
                    log_probs = log_probs.masked_fill(~label_mask, 0.0)
                    ret["log_probs"] = log_probs
                    return ret
```

### Expected behavior

Training does not perform temperature scaling on logits, which keeps it aligned with inference, making the calculated rollout_actor_probs_pearson_corr meaningful.

**Comment by Kirrito-k423** [comment]
本轮定时清理检查发现，这个 issue 最近一次非当前运行账号（Kirrito-k423）的公开回复或公开引用已经超过 6 个月，因此按社区 issue 定时清理标准关闭。

证据：
- latest_non_self_reply_or_reference_at: 2025-11-23T20:36:29Z
- 来源类型: cross-referenced, actor: JobQiu
- Issue: https://github.com/verl-project/verl/issues/4162
- Comments API: https://api.github.com/repos/verl-project/verl/issues/4162/comments
- Timeline API: https://api.github.com/repos/verl-project/verl/issues/4162/timeline
- Latest public item: https://github.com/verl-project/verl/issues/4162

如果作者或社区成员仍能提供新的复现、证据或需求背景，欢迎继续评论请求重开，或创建带有最新信息的新 issue。

English:
This scheduled triage found that the latest public reply or public reference from an account other than the current automation account (Kirrito-k423) is older than 6 months, so this issue is being closed under the scheduled stale-issue cleanup policy.

Evidence:
- latest_non_self_reply_or_reference_at: 2025-11-23T20:36:29Z
- Source type: cross-referenced, actor: JobQiu
- Issue: https://github.com/verl-project/verl/issues/4162
- Comments API: https://api.github.com/repos/verl-project/verl/issues/4162/comments
- Timeline API: https://api.github.com/repos/verl-project/verl/issues/4162/timeline
- Latest public item: https://github.com/verl-project/verl/issues/4162

If the author or another community member can provide a fresh reproduction, evidence, or updated requirement context, please comment to request reopening or open a new issue with the latest details.

This response was generated by a scheduled Codex GPT5.5 issue processing task.
